flashbang.hpp

```diff
@@ -0,0 +1,186 @@
+#include <stdbool.h>
+
+// class RGB;
+class LED;
+
+union UnionARGBType;
+
+class RGB {
```

Comment:
struct?

Reply:
no, as == is overloaded and we want a default constructor